### ISSUE TITLE
 vagrant provision crashes in the middle of many Sites (vhosts) because Apache can't bind to port 80 #1948 

### ISSUE THREAD

**Issue by svpernova09** [svpernova09/homestead#2]
Originally reported https://github.com/laravel/homestead/issues/1948

TL;DR

Multiple Apache sites cause port collisions.

**Comment by skeeith** [comment]
@svpernova09 does this bug still needs fixing?

**Comment by svpernova09** [comment]
> @svpernova09 does this bug still needs fixing?

Yes it does. I haven't had time to look into it but I believe it's still an issue. Feel free tot sir a shot at it and I'll review it as soon. As I can

**Comment by svpernova09** [comment]
Thanks ping me if you need anything
